backend/src/main/java/ca/bc/gov/hlth/hnweb/service/EligibilityService.java

```diff
@@ -78,16 +76,37 @@
 	 * @return The R15 response.
 	 * @throws HL7Exception
 	 */
-	public Message checkEligibility(R15 r15) throws HL7Exception {
+	public Message checkEligibility(R15 r15) throws HNWebException, HL7Exception {
 		String r15v2 = parser.encode(r15);
 
-		// TODO (weskubo-hni) Send to actual endpoint when it's available, currently
-		// response is stubbed.
-//		ResponseEntity<String> response = postCheckEligibilityRequest(r15v2, UUID.randomUUID().toString());
-//		String responseBody = response.getBody();
-		String responseBody = generateCannedResponse(r15);
+		ResponseEntity<String> response = postHibcRequest(r15Path, r15Username, r15Password, r15v2);
+		
+		if (response.getStatusCode() != HttpStatus.OK) {
+			logger.error("Could not connect to downstream service. Service returned {}", response.getStatusCode());
+			throw new HNWebException(ExceptionType.DOWNSTREAM_FAILURE);
+		}
+
+		return parseResponse(response.getBody(), "R15");
+	}
+	
+	/**
+	 * Checks for MSP Coverage Status based on the E45 request.
+	 * @param e45
+	 * @return
```

Comment:
Javadoc is inconsistent in this file
Some methods have Javadoc, some are missing @return tag descriptions like here
Some are missing @param and @throws tag descriptions.

Reply:
Updated javadoc for consistency.